server/src/main/java/org/candlepin/controller/OwnerManager.java

```diff
@@ -198,19 +226,18 @@ public void refreshContentAccessMode(OwnerServiceAdapter adapter, Owner owner) {
 
         // This shouldn't happen, but in the event our upstream source is having issues, let's
         // not put ourselves in a bad state as well.
-        if (upstreamList != null && !upstreamList.isEmpty()) {
+        if (!StringUtils.isBlank(upstreamList)) {
             // Not empty list. Verify that the upstream mode is present.
             String[] list = upstreamList.split(",");
-
-            if (!StringUtils.isEmpty(upstreamMode) && !ArrayUtils.contains(list, upstreamMode)) {
-                throw new IllegalStateException("Upstream access mode is not present in access mode list");
+            if (upstreamMode == null || !ArrayUtils.contains(list, upstreamMode)) {
```

Comment:
We are no longer allowing the owner fields for contentAccessMode and contentAccessModeList to be null or blank